File: src/main/java/net/kenro/ji/jin/purescript/psi/impl/PSDoNotationLetImpl.java
package net.kenro.ji.jin.purescript.psi.impl;

import com.intellij.lang.ASTNode;
import net.kenro.ji.jin.purescript.psi.PSDoNotationLet;

public class PSDoNotationLetImpl extends PSPsiElement implements PSDoNotationLet {

    public PSDoNotationLetImpl(final ASTNode node) {
        super(node);
    }

}

File: src/main/kotlin/org/purescript/psi/dostmt/PSDoNotationLetImpl.kt
package org.purescript.psi.dostmt

import com.intellij.lang.ASTNode
import org.purescript.psi.PSPsiElement
import org.purescript.psi.declaration.PSValueDeclaration

class PSDoNotationLet(node: ASTNode) : PSPsiElement(node) {
    val valueDeclarations get() =
        findChildrenByClass(PSValueDeclaration::class.java)
}